shared/src/main/scala/com/cibo/evilplot/numeric/BoxPlotSummaryStatistics.scala

```diff
@@ -42,7 +42,8 @@ case class BoxPlotSummaryStatistics(
   middleQuantile: Double,
   upperQuantile: Double,
   outliers: Seq[Double],
-  allPoints: Seq[Double]) {
+  allPoints: Seq[Double],
+  cluster: Int) {
```

Comment:
I feel as if this is just a container for the results of the box plot summary stuff, and it probably shouldn't contain the cluster number (I think there is a `BoxRendererContext` or similar that I'd be more comfortable using for stuff like this)